PWGHF/TableProducer/candidateSelectorDsToKKPi.cxx

```diff
@@ -280,10 +280,10 @@ struct HfCandidateSelectorDsToKKPi {
         }
         continue;
       }
-      if (pidDsToKKPi) {
+      if (TESTBIT(statusDsToKKPi, aod::SelectionStep::RecoTopol) && pidDsToKKPi) {
```

Comment:
```suggestion
      if (topolDsToKKPi && pidDsToKKPi) {
```
No need to check the bitmap here since we have directly the `bool` from the topological selection.

Reply:
Thank you @fcatalan92 for your suggestions! Indeed, we observed candidates which were passing the PID selections without passing the topological ones for a certain mass hypothesis. I have now implemented your suggestions